script/update

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# script/update: Update application to run for its current checkout.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+echo "==> Bootstrapping..."
+script/bootstrap
+
+echo "==> Running database migrations..."
+bundle exec rails db:migrate
+
+echo "==> Running test database migrations..."
+RAILS_ENV="test" bundle exec rails db:migrate
```

Comment:
Is this right? I thought the test database should use `bundle exec rails db:schema:load` as it'll truncate any static data added in the migrations?

Reply:
Yeah, you're probably right actually. I prefer not to add data in migrations, but this is probably safer